virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc

```diff
@@ -28,4 +28,9 @@ include::modules/virt-customizing-storage-profile.adoc[leveloffset=+1]
 
 == Additional resources
```

Comment:
You could add an anchor to this heading while you're in here if you'd like, per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies.

Reply:
Weird.... I had [id="additional-resources_creating-data-volumes-using-profiles"] in my copy of this file. There was a blank line between the ID and heading and I removed that.